docs/users/GettingStarted.md

```diff
@@ -1,26 +1,33 @@
 
 !!! warning "Important Notice"
-    The Nexus Mods app is still in development and is not recommend to be used as your primary mod manager. For the latest supporter mod manager please see [Vortex](https://staticdelivery.nexusmods.com/mods/2295/images/thumbnails/1/1-1690272155-1455020317.png).
+    The Nexus Mods app is still in development and is not recommend to be used as your primary mod manager. For the latest supported mod manager please see [Vortex](https://www.nexusmods.com/site/mods/1).
 
-Before installing the app, please ensure that your PC meets the system requirements for both [the App](./SystemRequirements.md) and the game you are modding.
+Before installing the app, please ensure that your PC meets the system requirements for both [this app](./SystemRequirements.md) and the game you are modding.
 
 ## Installation
 
-### Windows
+Download the latest version of the app and run the installer on your system.
 
-Download and run the [Windows Installer] from the [latest release] on GitHub.
+[:fontawesome-brands-windows: Windows Installer](https://github.com/Nexus-Mods/NexusMods.App/releases/latest/download/NexusMods.App.x64.exe){ .md-button .md-button--primary }
+[:fontawesome-brands-linux: Linux AppImage](https://github.com/Nexus-Mods/NexusMods.App/releases/latest){ .md-button }
```

Comment:
Should use forward defined links. The AppImage link can point to `https://github.com/Nexus-Mods/NexusMods.App/releases/latest/download/NexusMods.App.x86_64.AppImage`.

This should also mention that other version are available as well and link to the release itself at `https://github.com/Nexus-Mods/NexusMods.App/releases/latest`, as well as the repology table for system packages.

Reply:
I think you're getting bogged down a bit again. This is intended to provide the easiest possible way for users to get started. I've updated the AppImage link and added a link to "Other releases" below the CTA buttons but anything further is unnecessary.